search.py

```diff
@@ -0,0 +1,31 @@
+import web
+import model
+import utilities
+import json
+
+class Search:
+    def GET(self):
+        model.insert_view(web.ctx.host + web.ctx.fullpath, web.ctx.status,
+            web.ctx.ip, (web.ctx.env['HTTP_USER_AGENT'] if 'HTTP_USER_AGENT'
+                in web.ctx.env else None), "GET")
+        web.header('Content-Type', 'application/json')
+        web.header('Access-Control-Allow-Origin', '*')
+
+        data = web.input()
+
+        if "query" not in data:
+            return utilities.api_error("parameters must include \"query\"")
+
+        query = data.query
+        results = model.search(query, 30)
```

Comment:
do you think this endpoint returning 30 results is the best way to do this? what if we handle it like other endpoints, with a `limit` and `offset`?